### PR TITLE
Cookie

### PR DESCRIPTION
**Pull request by allanchau**
This PR adds a cookie library.

## Related PRs

- https://github.com/idearium/sso-lib/pull/13
- https://github.com/idearium/ras-rasnet/pull/248

### Testing

- [x] Check the tests.
- [x] Test via the related PR.

### CODE REVIEW

File: packages/cookie/tests/index.test.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const { create, find } = require('../');
+
+test('it can create cookies', () => {
+    const now = new Date();
+    const serialized = create({ cookie: 'cookie string', name: 'test' });
+
+    console.log('serialized', serialized, typeof serialized);
```
Comment (smebberson) [inline packages/cookie/tests/index.test.js:9]: @allanchau, can this be removed?

File: docusaurus/docs/cookie.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+-   `find` - finds a cookie in a HTTP cookie header string.
+
+### create
+
+To create a new cookie:
+
+```js
+const { create } = require('@idearium/cookie');
+
+await create({
```
Comment (smebberson) [inline docusaurus/docs/cookie.md:38]: @allanchau, why is `await` here? There are no promises involved?

File: docusaurus/docs/cookie.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+The create function uses the [cookie](https://www.npmjs.com/package/cookie#options-1) library in the background and can accept any `serialize` options listed there.
+
+### find
+
+To find a specific cookie within the cookie header:
+
+```js
+const { find } = require('@idearium/cookie');
+
+const cookie = await find({
```
Comment (smebberson) [inline docusaurus/docs/cookie.md:53]: @allanchau, same thing here regarding `await`?

File: docusaurus/docs/cookie.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+```js
+const { create } = require('@idearium/cookie');
+
+await create({
+    cookie: 'cookie string value',
+    name: 'foo',
+});
+```
+
+The create function uses the [cookie](https://www.npmjs.com/package/cookie#options-1) library in the background and can accept any `serialize` options listed there.
```
Comment (smebberson) [inline docusaurus/docs/cookie.md:44]: Below this, could you please do a little more complex example and set `httpOnly` and `secure` perhaps?

Comment (smebberson) [reply docusaurus/docs/cookie.md:44]: Also, I think a more complete example in some middleware using `res.set('set-cookie', cookieStr);` might be good.

**Review by smebberson — CHANGES_REQUESTED** [review]
@allanchau, looking good bit a few things to add here.

**Comment by allanchau** [comment]
@smebberson I've updated this now

**Review by smebberson — APPROVED** [review]
@allanchau, looks good, please launch.

**Pull request merged**